alexxsanya/epic-mail-api
# 164756631-project-setup

**What this PR do ?**
- file structure setup for the project.
- setting virtual env, github, travisci.
- settting up testing env.

**How to test this manually?**
- check the changed files 
- visit the chore description [link](https://www.pivotaltracker.com/story/show/164756631)

@bekeplar @emmnucon
